### PR TITLE
Update android-port.apt

### PR DESCRIPTION
**Pull request by smarek**
Clarify Android usage options on heavy referenced documentation page

### CODE REVIEW

File: src/site/apt/httpcomponents-client-4.5.x/android-port.apt [hunk 1]
```diff
@@ -49,6 +49,22 @@ HttpClient for Android
 
     Those users who want to continue using Apache HttpClient on Android are advised to 
     consider 
+    
+    * Apache HttpClient 4.5.x stock version, which works well with Android API 26 and newer
+
```
Comment (ok2c) [inline src/site/apt/httpcomponents-client-4.5.x/android-port.apt]: @smarek Unfortunately this is not entirely true. HttpClient 4.5 ships with `org.apache.http.conn.ssl.DefaultHostnameVerifier` class that depends on `javax.naming` APIs unsupported by Android.

https://github.com/ok2c/httpclient-android-ext/blob/master/httpclient-android/lint.xml

This class needs to be replaced with something else. I was going to add a reference to my project that provided such extensions as an option

https://ok2c.github.io/httpclient-android-ext/

Comment (smarek) [reply src/site/apt/httpcomponents-client-4.5.x/android-port.apt]: Well, that was purpose of the patchset and DistinguishedNameParser/DefaultHostnameVerifier patch

However I'd expect the HttpClient GA release to contain environment features detection and use the correct way. such as using parent class (ie. GeneralHostnameVerifier) which would choose the implementation based on environment, not requiring users to re-configure/provide custom components to official released httpclient

Comment (ok2c) [reply src/site/apt/httpcomponents-client-4.5.x/android-port.apt]: @smarek Not in the stable 4.5.x branch. In 5.0 (master) there is no need for any that as the `DefaultHostnameVerifier` works just fine as it.

Comment (smarek) [reply src/site/apt/httpcomponents-client-4.5.x/android-port.apt]: Oh, my confusion, is there a roadmap of 5.0 GA expected release date?

Comment (smarek) [reply src/site/apt/httpcomponents-client-4.5.x/android-port.apt]: Anyway, as far as i can tell, having my project published stable 4.5.x patched version is not bad, as it doesn't work with Android without modifications, but there is no need to release any 5.x version, which would make the repackaged library obsolete.

Comment (ok2c) [reply src/site/apt/httpcomponents-client-4.5.x/android-port.apt]: @smarek Pretty much so. I cannot commit to any particular time frame for 5.0 GA but Q3 2019 / Q4 2019 should be realistic.

Comment (ok2c) [reply src/site/apt/httpcomponents-client-4.5.x/android-port.apt]: @smarek Please let me know what I should do with this PR? Are you going to re-spin it or shall merge it as is?

**Comment by ok2c** [comment]
@smarek Could you please squash the commits into a single one?

**Comment by smarek** [comment]
@ok2c done :)

**Pull request merged**